### PR TITLE
[Accounts] Use enum for account names

### PR DESCRIPTION
**Pull request by avano**
//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: rest-tests/src/test/java/io/syndesis/qe/rest/tests/steps/connection/Connections.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public void createConnection(DataTable connectionProperties) {
     @Given("^create ActiveMQ connection$")
     public void createActiveMQConnection() {
         final List<List<String>> table = new ArrayList<>();
-        final Account account = accountsDirectory.getAccount("AMQ").get();
+        final Account account = accountsDirectory.getAccount(Account.Name.ACTIVEMQ.toString()).get();
```
Comment (asmigala) [inline rest-tests/src/test/java/io/syndesis/qe/rest/tests/steps/connection/Connections.java]: Can we add a `getAccount(Account.Name)` method to `AccountsDirectory`?

Comment (avano) [reply rest-tests/src/test/java/io/syndesis/qe/rest/tests/steps/connection/Connections.java]: :+1:

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**